tests/components/area-series-tests.js

```diff
@@ -0,0 +1,54 @@
+import test from 'tape';
+import React from 'react';
+import {mount} from 'enzyme';
+import XYPlot from 'plot/xy-plot';
+import AreaSeries from 'plot/series/area-series';
+import {testRenderWithProps, GENERIC_XYPLOT_SERIES_PROPS} from '../test-utils';
+import AreaChartElevated from '../../showcase/plot/area-chart-elevated';
```

Comment:
it doesn't look like it? when i try to i get a 

Error: Cannot find module '../showcase/plot/area-chart-elevated'